### PR TITLE
[tg-2794] fullscreen and close changes

### PR DESCRIPTION
**Pull request by ErikNijland**

### CODE REVIEW

File: e2e/close-straatbeeld.test.js [hunk 1]
```diff
@@ -3,18 +3,14 @@
 describe('Navigating to and away from straatbeeld', function () {
```
Comment (Vegter) [inline e2e/close-straatbeeld.test.js]: describe('The navigation to and ....

File: e2e/history-navigation-straatbeeld.test.js [hunk 2]
```diff
@@ -3,10 +3,6 @@
 describe('Navigating forwards and backwards through history at straatbeeld', function () {
```
Comment (Vegter) [inline e2e/history-navigation-straatbeeld.test.js]: The back- and forward navigation ...

File: e2e/homepage.test.js [hunk 3]
```diff
@@ -1,5 +1,5 @@
 describe('The homepage', () => {
-    describe('navigate to the map w/ layer selection openend', () => {
+    describe('navigate to the map w/ layer selection opened', () => {
```
Comment (Vegter) [inline e2e/homepage.test.js]: Na describe('The homepage
verwacht je hier describe('The navigation to...

File: e2e/homepage.test.js [hunk 4]
```diff
@@ -1,5 +1,5 @@
...
         it('close layer selection, then the map, return to the homepage', () => {
```
Comment (Vegter) [inline e2e/homepage.test.js]: it('closes... and then returns...

File: e2e/homepage.test.js [hunk 5]
```diff
@@ -16,34 +16,26 @@ describe('The homepage', () => {
             dp.validate('PAGE--HOME', page);
         });
 
-        it('make the map fullscreen, close it, return to the homepage', () => {
+        it('minimize the map, return to the homepage', () => {
```
Comment (Vegter) [inline e2e/homepage.test.js]: it('minimizes the map and returns to...

File: e2e/homepage.test.js [hunk 6]
```diff
@@ -16,34 +16,26 @@ describe('The homepage', () => {
...
             dp.validate('PAGE--HOME', page);
         });
     });
 
-    describe('navigate to straatbeeld', () => {
-        beforeEach(function () {
-            dp.storage.clearAll();
-        });
-
+    describe('navigate to straatbeeld w/ a map', () => {
```
Comment (Vegter) [inline e2e/homepage.test.js]: describe('The straatbeeld with a map navigation

File: e2e/homepage.test.js [hunk 7]
```diff
@@ -16,34 +16,26 @@ describe('The homepage', () => {
...
         it('and back to the homepage by closing it', () => {
```
Comment (Vegter) [inline e2e/homepage.test.js]: it(what?

File: modules/atlas/services/redux/reducers/map-reducers.factory.test.js [hunk 8]
```diff
@@ -251,15 +251,21 @@ describe('The map reducers', function () {
             expect(output.map.isFullscreen).toBe(false);
         });
 
-        it('when enabling fullscreen, the layer selection will be disabled', function () {
-            var inputState = angular.copy(DEFAULT_STATE),
+        it('when changing isFullscreen, the layer selection will also be disabled', function () {
```
Comment (Vegter) [inline modules/atlas/services/redux/reducers/map-reducers.factory.test.js]: it('disables layer selection hen changing fullscreen

File: modules/page/components/homepage/homepage.component.js [hunk 9]
```diff
@@ -9,27 +9,16 @@
...
-            };
-        }
-
-        $rootScope.$on('$destroy', deregistrationFn);
+        vm.fetchStraatbeeldPayload = {
+            id: 'TMX7315120208-000073_pano_0005_000451',
+            heading: 226,
+            isInitial: true,
+            isFullscreen: false
+        };
```
Comment (Vegter) [inline modules/page/components/homepage/homepage.component.js]: HOMEPAGE_CONFIG constants?

**Comment by ErikNijland** [comment]
Feedback verwerkt

**Pull request merged**